src/lib/models/types/reference.ts

```diff
@@ -115,10 +115,15 @@ export class ReferenceType extends Type
      * Return a string representation of this type.
      */
```

Comment:
You're welcome. I don't think there is currently a test set up. The travis build seems to have run fine.